### ISSUE TITLE
Need a download link to the SqlLocalDb utility

### ISSUE THREAD

**Issue by CLIRResearch** [MicrosoftDocs/sql-docs#5723]
Badly need a download link for this utility.  

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 80ae2735-1406-baf7-df41-5f2b930c132a
* Version Independent ID: 99241f21-4b3a-5a5d-1257-38ed9d5be434
* Content: [SqlLocalDB Utility - SQL Server](https://docs.microsoft.com/en-us/sql/tools/sqllocaldb-utility?view=sql-server-ver15)
* Content Source: [docs/tools/sqllocaldb-utility.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/tools/sqllocaldb-utility.md)
* Product: **sql**
* Technology: **tools-other**
* GitHub Login: @markingmyname
* Microsoft Alias: **maghan**

**Comment by WilliamAntonRohm** [comment]
@CLIRResearch -- thank you for your feedback.  It appears SqlLocalDb.exe is included with SQL Server Express:

- [SQL Server Express LocalDB](https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/sql-server-express-localdb?view=sql-server-ver15)

The utility location should be:
- `C:\Program Files\Microsoft SQL Server\nnn\Tools\Binn\SQLLocalDB.exe`

Where _nnn_ is the current SQL Server edition number.

Please also consider these resources:
- [SQL Server on Microsoft Q&A](https://docs.microsoft.com/en-us/answers/products/sql-server)
- [DBA Stack Exchange](https://dba.stackexchange.com/questions/tagged/sql-server)
- [Stack Overflow](https://stackoverflow.com/questions/tagged/sql-server) 


I am closing this issue now.  You are welcome to @ mention me for any followup.
We hope to hear from you again.

**Comment by NikitaPozharskyi** [comment]
@WilliamAntonRohm Hey!
with latest version of MSSQL (16.xx) I cannot find this utility using this famous path C:\Program Files\Microsoft SQL Server\nnn\Tools\Binn\SQLLocalDB.exe
SQL Server 2022 express edition was installed via link you've provided below. Just to clarify before that I installed MSSQL server 2022 Developer edition so I have installed both but cannot find this utility. I've tried to find it on my pc after the installation using the search bar in windows and explorer but unfortunately nothing helped. 
Recently I was using the 2019 MSSQL server and I had this one on my pc but Security department forced me to update on never version and I am having issues now because custom local scripts was written with sqllocaldb utility
